spec/systems/users_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe User, type: :system do
+  describe 'Signup feature', :aggregate_failures do
```

Comment:
What is `:aggregate_failures`?

Reply:
`Allows multiple expectations in the provided block to fail, and then aggregates them into a single exception, rather than aborting on the first expectation failure like normal. This allows you to see all failures from an entire set of expectations without splitting each off into its own example (which may slow things down if the example setup is expensive).`

https://www.rubydoc.info/github/rspec/rspec-expectations/RSpec%2FMatchers%3Aaggregate_failures
https://relishapp.com/rspec/rspec-core/docs/expectation-framework-integration/aggregating-failures

Reply:
But should we use this option on CI? I think fail-fast is better because we can reduce the testing time

Reply:
Good catch. I will make the change.

Reply:
Fixed on 3b32d4e07a5324274a0aaf259b727f55f581ef59